admin-ui/src/Meals/service.ts

```diff
@@ -0,0 +1,77 @@
+
+import { ApolloClient, gql } from '@apollo/client';
+
+const searchStringQuery = gql`
+  query SearchStrings($searchString: String!) {
+    query {
+      meals(
+        filter: {
+          or: [
+            { code: { includes: $searchString } }
+            { nameEn: { includes: $searchString } }
+            { nameFr: { includes: $searchString } }
+            { descriptionEn: { includes: $searchString } }
+            { descriptionFr: { includes: $searchString } }
+            { tags: { contains: [$searchString] } }
+          ]
+        }
+      ) {
+        edges {
+          node {
+            rowId
+          }
+        }
+      }
+    }
+  }
+`;
+
+const searchStringCategoryQuery = gql`
+  query searchCategory($searchString: CategoryT!) {
+    query {
+      meals(filter: { categories: { anyEqualTo: $searchString } }) {
+        edges {
+          node {
+            rowId
+          }
+        }
+      }
+    }
+  }
+`;
+
+const extractIdsFromResult = (result: any) => {
```

Comment:
Please look at the other PR #506 for types related comments and please make those changes

Reply:
done as in the PR[#505](https://github.com/CivicTechFredericton/mealplanner/issues/505). thank you again